manifests/ntpd.pp

```diff
@@ -0,0 +1,19 @@
+# Configure NTP daemon
+class foreman_network::ntpd {
+  if ($foreman_network::configure_ntpd) {
+
+    exec { 'NTP sync':
```

Comment:
I don't think that actually runs ntpdate. I'm also not really convinced that module is a great benefit so I'm thinking:
```puppet
package { ['ntp', 'ntpdate']:
  ensure => installed,
} ~>
exec { 'NTP sync':
  command     => "/sbin/service ntpd stop; /usr/sbin/ntpdate ${foreman_network::ntp_host}",
  refreshonly => true,
} ~>
service { 'ntpd':
  ensure => running,
  enable => true,
}
```
/cc @domcleal